target-riscv/cpu.h

```diff
@@ -80,6 +80,7 @@ struct CPURISCVState {
     uint32_t amoinsn;
     target_long amoaddr;
     target_long amotest;
+    target_long rve;
```

Comment:
Hi Kito,

rve should be in tb_flags, as proposed by Stefan in #59(see the discussion there why extension ought to be in tb_flags), which means this should be merged on top of Stefan's pull. After his last issue is resolved, I think we can merge it. Other than that it looks good to me, thanks!

Cheers,
Bastian